OPHD/UI/StructureListBox.cpp

```diff
@@ -41,10 +41,10 @@ static void drawItem(Renderer& renderer, StructureListBox::StructureListBoxItem&
 
 
 StructureListBox::StructureListBoxItem::StructureListBoxItem(Structure* s) :
-	ListBoxItem(s->name()),
-	structure(s),
-	structureState(""),
-	colorIndex(s->state())
+	ListBoxItem{s->name()},
```

Comment:
What's the benefit of this change?

Reply:
Ahh, so I've been using the `{}` style in most other parts of the project, so partly that was for consistency, though there are some benefits to the newer syntax.

The brace init list `{}` works with either a constructor, or with a `struct` which has no constructor, where the values would then be taken to initialize the fields of the struct, in order. The function call init style `()` will only work with a constructor. If all the constructor does is set struct fields, in order, then changing the init style to `{}` allows you to potentially move the explicit constructor.

I believe I was planning to convert the `ListBoxItem` structs to be plain `struct` without a constructor. I don't see such a change in this particular change set, though I believe that was something I was moving towards when I made that change.

Reply:
Works for me, thanks!